### ISSUE TITLE
[Postgresql] ARM support

### ISSUE THREAD

**Issue by mgonzaga1990** [kubedb/project#779]
I'm new to kubedb and would like to find out if i can use this to build a cluster of postgre database in my raspberry pi using kubernetes (microk8s) so far i have tried the examples but didn't get any good results.

Wanna check if there is or has anyone tried to set this up in raspberry pi.

Thanks,

**Comment by stenwt** [comment]
The operator runs, with version 0.13 on. I was able to run the operator with my k3s cluster on Pi4s: 

```helm template --namespace kube-system --release-name kubedb --set nodeSelector="" --version v0.13.0-rc.0 appscode/kubedb  | kubectl apply -f```

However, the kubedb database images are still x86_64 only, so the managed databases fail with:

**Comment by mgonzaga1990** [comment]
yeah.. so i ended up creating my own image that could do master - slave in my k8s cluster
